src/guide/list.md

```diff
@@ -268,25 +268,23 @@ Similar to template `v-if`, you can also use a `<template>` tag with `v-for` to
 Note that it's **not** recommended to use `v-if` and `v-for` together. Refer to [style guide](../style-guide/#avoid-v-if-with-v-for-essential) for details.
 :::
 
-When they exist on the same node, `v-for` has a higher priority than `v-if`. That means the `v-if` will be run on each iteration of the loop separately. This can be useful when you want to render nodes for only _some_ items, like below:
+When they exist on the same node, `v-if` has a higher priority than `v-for`. That means that if `v-if` condition depends on the property from `v-for` scope, we will have an error: 
```

Comment:
I think I'd avoid using the word 'property' to describe the iteration variable. The error message uses the word 'property' but that's because it's looking for a property on the instance.

Perhaps something like:

```text
That means the `v-if` condition will not have access to variables from the scope of the `v-for`.
```

Reply:
I don't think we should remove this line altogether because we need to show that this code snippet is incorrect. I changed a message to reflect this better